bindings/python/examples/dnn/dnn.py

```diff
@@ -93,19 +93,25 @@ class ModesEnum(Enum):
     if not status:
         print("system.getAvailableModes() failed with status: ", status)
 
+    status = cameras[0].setMode(modes[ModesEnum.MODE_NEAR.value])
+    if not status:
+        print("cameras[0].setMode() failed with status: ", status)
+    
+    cameras[0].setControl("loadModuleData", "call")
+
     types = []
     status = cameras[0].getAvailableFrameTypes(types)
     if not status:
         print("system.getAvailableFrameTypes() failed with status: ", status)
-
-    status = cameras[0].setFrameType(types[0])
```

Comment:
types[0] is 'lt_bin' frame type. This type returns black frame image. Hence, setting the frame type as 'mp_pcm'

Reply:
You can use the exact name of the frame type instead on relying on position 2 in the list.
e.g. setFrameType("mp_pcm")